### ISSUE TITLE
Error while installing Debugger (openSUSE Leap)

### ISSUE THREAD

**Issue by bdbai** [OmniSharp/omnisharp-vscode#510]
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (1.0.0-preview2-003121)

Product Information:
 Version:            1.0.0-preview2-003121
 Commit SHA-1 hash:  1e9d529bc5

Runtime Environment:
 OS Name:     opensuse
 OS Version:  42.1
 OS Platform: Linux
 RID:         opensuse.42.1-x64
```
VS Code version: `1.2.1` `fe7f407b95b7f78405846188259504b34ef72761`
C# Extension version: `1.2.0`

## Steps to reproduce
Launch VS Code and enter a C# project generated by `dotnet new`.

## Expected  behavior
No error output and start debugging after pressing F5.

## Actual behavior
VS Code says `Error while installing .NET Core Debugger.`. Here is the log: [coreclr-debug](https://gist.github.com/bdbai/8be34b9e41e7d40b5d68a7703ad19cb2)
Code linting and Intellisense works well. `dotnet build && dotnet run` in terminal works well too. But when I hit F5, VS Code complaints `Debug adapter executable '/home/bdbai/.vscode/extensions/ms-vscode.csharp-1.2.0/coreclr-debug/debugAdapters/OpenDebugAD7' not found.`
Any workarounds there?

**Comment by gregg-miskelly** [comment]
@chuckries The issue is that we are using the rid from `dotnet --info` which is `opensuse.42.1-x64`. But the rid that we need to be passing is `opensuse.13.2-x64`. Maybe we should stop using `dotnet --info` to find the rid and instead use the same code as we use for OmniSharp?

**Comment by DustinCampbell** [comment]
Note that we already have the distro sniffing code to do this for OmniSharp downloads. You could leverage that.

**Comment by bdbai** [comment]
Thanks for response. I've tried OmniSharp 1.2.1, which still fails as before. Could you please point out the commit that fixes it?

This workaround seems to work:

```diff
diff --git a/src/coreclr-debug/activate.ts b/src/coreclr-debug/activate.ts
index 77c0c8b..877af95 100644
--- a/src/coreclr-debug/activate.ts
+++ b/src/coreclr-debug/activate.ts
@@ -190,5 +190,10 @@ function getDotnetRuntimeId(): string {
         throw new Error('Unable to determine Runtime ID');
     }
 
+    if (rid.startsWith('opensuse.42')) {
+        _util.log('Warning: Use opensuse.13.2 instead.');
+        return 'opensuse.13.2';
+    }
+
     return rid;
 }
diff --git a/src/coreclr-debug/install.ts b/src/coreclr-debug/install.ts
index be06f4f..f62a45c 100644
--- a/src/coreclr-debug/install.ts
+++ b/src/coreclr-debug/install.ts
@@ -102,6 +102,11 @@ export class DebugInstaller
                 src += '.exe';
                 dest += '.exe';
             }
+            if (CoreClrDebugUtil.existsSync(src + '.dll')) {
+                src += '.dll';
+                dest += '.dll';
+            }
+ 
         }
 
         const promise = new Promise<void>((resolve, reject) => {
@@ -187,6 +192,12 @@ export class DebugInstaller
             },
             frameworks: {
                 "netcoreapp1.0": {
+                    "dependencies": {
+                      "Microsoft.NETCore.App": {
+                        "type": "platform",
+                        "version": "1.0.0"
+                      }
+                    },
                     imports: [ "dnxcore50", "portable-net45+win8" ]
                  }
             },
@@ -228,4 +239,4 @@ export class DebugInstaller
 
         return promise;
     }
-}
\ No newline at end of file
+}
```

**Comment by gregg-miskelly** [comment]
Correct, this hasn't been fixed yet. Dustin was just telling Chuck how he could fix this.

**Comment by bdbai** [comment]
Oops I failed to read carefully. Sorry for that.

**Comment by chuckries** [comment]
This is fixed and will be available in the 1.3 release.
